Maple2.Database/Model/Map/CubeHelper.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Maple2.Model.Enum;
+using Maple2.Model.Game;
+
+namespace Maple2.Database.Model;
+
+internal static class CubeHelper {
+    public static CubeSettings? GetCubeSettings(PlotCube cube) {
```

Comment:
Do we really need this? Or can we just do the conversion within the model? Similar to how ItemAppearance is done.